cpu/native/periph/i2c_mock.c

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (C) 2024 COGIP Robotics association
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     cpu_native
+ * @ingroup     drivers_periph_i2c
+ * @{
+ *
+ * @file
+ * @brief       empty I2C implementation
```

Comment:
```suggestion
 * @brief       Pluggable no-op I2C implementation
 *
 * By default, operations on this bus will always succeed (without actually performing any
 * action). Reads will produce zeros.
 *
 * To provide mocks of a particular connected device, application authors can provide any
 * of the functions themselves, because all the I2C functions in this implementation are
  * weak symbols. There is per-bus or per-address dispatch. That means that if multiple
  * buses or devices are to be mocked with non-zero implementations, the application needs
  * to provide a single implementation that fans out to the respective mocked device.
```